services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/application.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from fastapi import FastAPI
+
+from .api import main_router
+from .models import ApplicationHealth
+from .remote_debug import setup as remote_debug_setup
+from .settings import ServiceSidecarSettings
+from .shared_handlers import on_shutdown_handler
+from .storage import SharedStore
+
+logger = logging.getLogger(__name__)
+
+
+def assemble_application() -> FastAPI:
+    """
+    Creates the application from using the env vars as a context
+    Also stores inside the state all instances of classes
+    needed in other requests and used to share data.
+    """
+
+    dynamic_sidecar_settings = ServiceSidecarSettings.create()
+
+    logging.basicConfig(level=dynamic_sidecar_settings.loglevel)
+    logging.root.setLevel(dynamic_sidecar_settings.loglevel)
+    logger.debug(dynamic_sidecar_settings.json(indent=2))
+
+    application = FastAPI(debug=dynamic_sidecar_settings.debug)
```

Comment:
For uniformity, please initialize the app with the same doc entrypoints and options as the other services (see [here](https://github.com/ITISFoundation/osparc-simcore/blob/524137aac07093cb886f6f9236e61b3bd60271ff/services/api-server/src/simcore_service_api_server/core/application.py#L42). 

- ``/dev/doc`` should be the swagger doc entrypoint

Reply:
done, also added the api "v1" tags in the routes as they were missing.